src/makielayout/blocks/axis.jl

```diff
@@ -281,118 +340,283 @@ function initialize_block!(ax::Axis; palette = nothing)
-        ticklabelspace = ax.yticklabelspace, ticks = ax.yticks, tickformat = ax.ytickformat, ticklabelsvisible = ax.yticklabelsvisible,
-        ticksvisible = ax.yticksvisible, spinevisible = yspinevisible, spinecolor = yspinecolor, spinewidth = ax.spinewidth,
-        trimspine = ax.ytrimspine, ticklabelsize = ax.yticklabelsize, ticksize = ax.yticksize, flip_vertical_label = ax.flip_ylabel, reversed = ax.yreversed, tickwidth = ax.ytickwidth,
-            tickcolor = ax.ytickcolor,
-        minorticksvisible = ax.yminorticksvisible, minortickalign = ax.yminortickalign, minorticksize = ax.yminorticksize, minortickwidth = ax.yminortickwidth, minortickcolor = ax.yminortickcolor, minorticks = ax.yminorticks, scale = ax.yscale,
-        )
+    yaxis = LineAxis(
+        topscene,
+        endpoints = yaxis_endpoints,
+        limits = ylims,
+        flipped = yaxis_flipped,
+        ticklabelrotation = ax.yticklabelrotation,
+        ticklabelalign = ax.yticklabelalign,
+        labelsize = ax.ylabelsize,
+        labelpadding = ax.ylabelpadding,
+        ticklabelpad = ax.yticklabelpad,
+        labelvisible = ax.ylabelvisible,
+        label = ax.ylabel,
+        labelfont = ax.ylabelfont,
+        labelrotation = ax.ylabelrotation,
+        ticklabelfont = ax.yticklabelfont,
+        ticklabelcolor = ax.yticklabelcolor,
+        labelcolor = ax.ylabelcolor,
+        tickalign = ax.ytickalign,
+        ticklabelspace = ax.yticklabelspace,
+        ticks = ax.yticks,
+        tickformat = ax.ytickformat,
+        ticklabelsvisible = ax.yticklabelsvisible,
+        ticksvisible = ax.yticksvisible,
+        spinevisible = yspinevisible,
+        spinecolor = yspinecolor,
+        spinewidth = ax.spinewidth,
+        trimspine = ax.ytrimspine,
+        ticklabelsize = ax.yticklabelsize,
+        ticksize = ax.yticksize,
+        flip_vertical_label = ax.flip_ylabel,
+        reversed = ax.yreversed,
+        tickwidth = ax.ytickwidth,
+        tickcolor = ax.ytickcolor,
+        minorticksvisible = ax.yminorticksvisible,
+        minortickalign = ax.yminortickalign,
+        minorticksize = ax.yminorticksize,
+        minortickwidth = ax.yminortickwidth,
+        minortickcolor = ax.yminortickcolor,
+        minorticks = ax.yminorticks,
+        scale = ax.yscale,
+    )
 
     ax.yaxis = yaxis
 
-    xoppositelinepoints = lift(scene.px_area, ax.spinewidth, ax.xaxisposition; ignore_equal_values=true) do r, sw, xaxpos
-        if xaxpos == :top
-            y = bottom(r)
-            p1 = Point2f(left(r) - 0.5sw, y)
-            p2 = Point2f(right(r) + 0.5sw, y)
-            return [p1, p2]
-        else
-            y = top(r)
-            p1 = Point2f(left(r) - 0.5sw, y)
-            p2 = Point2f(right(r) + 0.5sw, y)
-            return [p1, p2]
+    xoppositelinepoints =
+        lift(scene.px_area, ax.spinewidth, ax.xaxisposition; ignore_equal_values = true) do r, sw, xaxpos
```

Comment:
These line breaks are a bit weird to me. Shouldn't this become 
```julia
... = lift(
    ...
) do ...
    ...
end
```
if it overflows?

Reply:
That's a bit weird yes.